core/feconf.py

```diff
@@ -423,6 +423,8 @@ class ClassifierDict(TypedDict):
 # Whether new explorations should have correctness-feedback enabled
 # by default.
 DEFAULT_CORRECTNESS_FEEDBACK_ENABLED = True
+# Whether edits can be made to a new exploration.
+DEFAULT_EDITS_ALLOWED = True
```

Comment:
I don't think this should be a constant here. Just inline it. (I can't see the point of the alternative -- why would we create an exploration and not allow it to be edited?)